### ISSUE TITLE
[$10][MSFT-21] [Challenge Feed] Challenge icon contrast between foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2744]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. In 'My Challenges' list; check the challenge icon (eg: Cd, F2F, Wb, DF2F, FE, Wg, MM)

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #3F8300. New contrast ratio: 4.52
- Passes AA, if you change text color to #424253. New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 2.08 (foreground color: #fafafb, background color: #60c700, font size: 9.8pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F6

**Comment by ThomasKranitsas** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by romitgit** [comment]
Please assign this to me.

**Comment by cwdcwd** [comment]
@romitgit, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
@romitgit, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
I'm not able to assign this.

**Comment by romitgit** [comment]
Please help.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@romitgit I see you removed your assignment from this ticket, please confirm if you are going to work on this or want us to mark this `OpenForPickup`. Thanks

**Comment by romitgit** [comment]
Done with changes. This is fixed in https://github.com/topcoder-platform/community-app/pull/2993

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2744 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097454 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097454 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue : https://github.com/topcoder-platform/community-app/pull/3062

Dependent PR
PR for topcoder-react-ui-kit : https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/15

**Comment by codeMinter** [comment]
@veshu @sushilshinde @nkumar-topcoder we fixed the color-palette in react-ui which takes care of this issue, so we don't need the PR for this. Please take this issue ahead for QA testing.

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097454<br/><br/>```This is an automated message for crazyk via Topcoder X```
